demo/api/src/news/entities/news.entity.ts

```diff
@@ -48,7 +47,6 @@ export class NewsContentScope {
 }
 
 @EntityInfo<News>((news) => ({ name: news.title, secondaryInformation: news.slug }))
-@RootBlockEntity()
```

Comment:
Why do you remove this here?

Reply:
sorry that was unintentional, added again here: a7240b9b2410feeccfb3e35855dcb241bc1a2431